packages/jaeger-ui/package.json

```diff
@@ -42,7 +42,7 @@
   },
   "dependencies": {
     "@jaegertracing/plexus": "0.2.0",
-    "@pyroscope/flamegraph": "0.18.4-1379-557f0b4.0",
+    "@pyroscope/flamegraph": "0.18.4",
```

Comment:
while I have you, I wanted to ask - in the last merge I am seeing that the numbers in the flame graph & table are in "units", which is pretty confusing. It sounds like a technical reason (library may not know the units of numbers), but couldn't we pass some parameter to it to make it more obvious?  Also, I'm just curious under what conditions the numbers in the flamegraph are not time units - if they are always time than it seems like the graph could always display them with proper time units like s, ms, etc.

Reply:
The flamegraph component can be used for visualizing other arbitrary data that does have [other units](https://github.com/pyroscope-io/pyroscope/blob/main/packages/pyroscope-flamegraph/src/format/format.ts#L19-L35).

Makes sense though that for traces it will always be time so made a Pr to fix this:
https://github.com/pyroscope-io/pyroscope/pull/1387

It's available in [v0.18.5](https://www.npmjs.com/package/@pyroscope/flamegraph)